source/_integrations/nibe_heatpump.markdown

```diff
@@ -47,6 +47,9 @@ Supported devices:
 To communicate with the pump there are a few different connection methods available depending on hardware. At the moment the following methods are supported.
 
 - [UDP Gateway using NibeGw](#udp-gateway-using-nibegw)
+- [ESPHome](#ESPHome)
+- [Arduino](#Arduino)
+- [Raspberry Pi-Linux](#Raspberry-Pi-Linux)
```

Comment:
All of these are sub solutions to udp gateway

Reply:
Ah, My bad, I misread the flow ;-)